apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -226,6 +226,9 @@ function ListBoxInline({ options, layout }) {
   const handleShowSearch = () => {
     const newValue = !showSearch;
     setShowSearch(newValue);
+    if (!isPopover && !selections.isActive()) {
```

Comment:
maybe also check for if it show or hide search?
(probably work OK without as I think for close it will already be in selection and the exit selections is done later)
```suggestion
    if (newValue && !isPopover && !selections.isActive()) {
```